internal/deno.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.225.0",
   "exports": {
     ".": "./mod.ts",
-    "./diff": "./diff.ts",
```

Comment:
now `diff`, `diffstr`, and `buildMessage` are only available from `@std/internal` (root). I think that would be inconvenient in the future when we added completely unrelated APIs to `@std/internal`